internal/repository/todo_repository.go

```diff
@@ -0,0 +1,131 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"final-project/internal/entity"
+)
+
+type TodoRepository struct {
+	db *sql.DB
+}
+
+func New(db *sql.DB) *TodoRepository {
+	return &TodoRepository{db: db}
+}
+
+func (t *TodoRepository) DeleteTodo(ctx context.Context, id int64) error {
+	res, err := t.db.ExecContext(ctx, `DELETE FROM scheduler WHERE id = :id`, sql.Named("id", id))
+	if err != nil {
+		return err
+	}
+
+	count, err := res.RowsAffected()
+	if err != nil {
+		return err
```

Comment:
Лучше все ошибки оборачивать, чтобы быстрее понимать, где произошла ошибка
```suggestion
		return fmt.Errorf("error in getting affected rows: %w", err)
```